lr-cli/cortx_setup/commands/__init__.py

```diff
@@ -85,5 +85,6 @@
     'PrepareConfstore',
     'PillarSync',
     'SetConfiguration',
-    'GetConfiguration'
+    'GetConfiguration',
+    'NodeResetCluster'
```

Comment:
Change section name to `ClusterResetNode`

Reply:
Done